### PR TITLE
delete outer resizable div along with inner element

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Interpret clicks to resizable div as clicks on inner child element.

Deleting the inner child element also deletes outer resizable div.

### CODE REVIEW

File: apps/src/applab/designMode.js [hunk 1]
```diff
@@ -34,6 +34,11 @@ designMode.onDivApplabClick = function (event) {
   if (element.id === 'divApplab') {
     designMode.clearProperties();
   } else {
+    if ($(element).is('.ui-resizable')) {
+      element = element.children[0];
+    } else if ($(element).is('.ui-resizable-handle')) {
+      element = element.parentNode.children[0];
+    }
```
Comment (Bjvanminnen) [inline apps/src/applab/designMode.js]: Not sure if there's a better way of looking for this than class name?

Comment (joshlory) [inline apps/src/applab/designMode.js]: Maybe `.data('uiResizable')`?

File: apps/src/applab/designMode.js [hunk 2]
```diff
@@ -220,10 +225,19 @@ designMode.onDonePropertiesButton = function() {
...
-  element.parentNode.removeChild(element);
-  if ($(element).hasClass('screen')) {
+  var isScreen = $(element).hasClass('screen');
+  var toRemove = element;
+  var parent = element.parentNode;
+  if ($(parent).is('.ui-resizable')) {
+    toRemove = parent;
+    parent = parent.parentNode;
+  }
+  parent.removeChild(toRemove);
```
Comment (joshlory) [inline apps/src/applab/designMode.js]: Nit: jQuery `.remove()` may be a little more readable (you don't need to pass in the parent, just call it on the node you want to remove).

File: apps/src/applab/designMode.js (hunk 1 above)
Comment (Bjvanminnen) [inline apps/src/applab/designMode.js]: Looks like that works for the parent (ui-resizable) but not for the handle element. Might stick with using class name for consistency across both.

File: apps/src/applab/designMode.js [hunk 3]
```diff
@@ -34,6 +34,11 @@ designMode.onDivApplabClick = function (event) {
   if (element.id === 'divApplab') {
     designMode.clearProperties();
   } else {
+    if ($(element).is('.ui-resizable')) {
+      element = element.children[0];
```
Comment (joshlory) [inline apps/src/applab/designMode.js]: We also make the "first child" assumption down in `makeUndraggable` -- maybe good to combine both into a function?
```javascript
var elm = $(':first-child', wrapper);
```

**Comment by joshlory** [comment]
A few small nits, but LGTM.  Yay for tests.  Thanks for fixing!

**Pull request merged**